arviz/plots/backends/matplotlib/distplot.py

```diff
@@ -86,20 +93,49 @@ def plot_dist(
     return ax
 
 
-def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs):
+def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs, is_circular):
     """Add a histogram for the data to the axes."""
+    bins = hist_kwargs.pop("bins")
```

Comment:
I think using `bins = hist_kwargs.pop("bins", None)` will allow deleting the `setdefault` (see comment below)